packages/plugin-webpack/plugin.js

```diff
@@ -54,28 +58,38 @@ module.exports = function plugin(config, args) {
         extendConfig = (cfg) => ({ ...cfg, ...args.extendConfig });
       }
 
-      let dom = new JSDOM(
-        fs.readFileSync(path.join(srcDirectory, config.devOptions.fallback))
-      );
+      // Get all html files from the output folder
+      const pattern = srcDirectory + "/**/*.html";
+      const htmlFiles = (await globAsync(pattern))
```

Comment:
nit: since we’re waiting on the scanning here without any heavy operations, what about switching this to `glob.sync()` and removing `util`? That’d save adding that dependency.

Reply:
`util` is a node built-in, but you're right - there's no need to do this async. I've updated it to `glob.sync`.